src/main/java/de/unijena/cheminf/fragment/fingerprint/performanceTest/PerformanceTest.java

```diff
@@ -0,0 +1,419 @@
+            FileWriter tmpBitFingerprintsResultWriter = new FileWriter(tmpBitFingerprintsResultFile, false);
+            this.bitPrintWriter = new PrintWriter(tmpBitFingerprintsResultWriter);
+            // count fingerprints process file
+            File tmpCountFingerprintsResultFile = new File(this.workingPath + "/Results/" + PerformanceTest.CSV_COUNT_FINGERPRINT_PROCESS_RESULT_FILE_NAME + tmpProcessingTime + ".csv");
+            FileWriter tmpCountFingerprintResultWriter = new FileWriter(tmpCountFingerprintsResultFile, false);
+            this.countPrintWriter = new PrintWriter(tmpCountFingerprintResultWriter);
+            // bit array file
+            File tmpBitArrayFingerprintResultFile = new File(this.workingPath + "/Results/" + PerformanceTest.BIT_ARRAY_RESULT_FILE_NAME + tmpProcessingTime+ ".txt");
+            FileWriter tmpBitArrayFingerprintResultWriter = new FileWriter(tmpBitArrayFingerprintResultFile, false);
+            this.bitArrayPrintWriter = new PrintWriter(tmpBitArrayFingerprintResultWriter);
+            // read in CSV files that contains fragments
+            try {
+                this.importDataFromTextFile();
+            } catch (IOException anException) {
+                this.exceptionsPrintWriter.println("Fragment load ERROR. Unsuitable (structure) CSV files were tried to be read in.");
+                this.exceptionsPrintWriter.flush();
+                this.appendToLogfile(anException);
+                throw new Exception("Fragment load ERROR. Unsuitable (structure) CSV files were tried to be read in. ");
+            }
+            // generate bit and count fingerprints
+            this.generateFingerprints(Integer.parseInt(anArgs[3]),1,2); // TODO delete arguments
+            this.resultsPrintWriter.flush();
+            this.bitPrintWriter.println();
+            this.bitPrintWriter.flush();
+            this.countPrintWriter.flush();
+            this.bitArrayPrintWriter.flush();
+            this.exceptionsPrintWriter.flush();
+        } catch (Exception anException) {
+            this.appendToLogfile(anException);
+            anException.printStackTrace(System.err);
+            System.exit(1);
+        }
+    }
+    //</editor-fold>
+    //
+    //<editor-fold defaultstate="collapsed" desc="Private methods">
+    /**
+     * The two text files (fragment and molecule file)
+     * are read in and the data provided so that in the next step the
+     * fingerprints can be calculated based on the information in these text files.
+     * For the method to work properly, it is important that the two files are in a specific form predicted
+     * for the method. Thus, each line in the fragment file corresponds to a fragment. Components
+     * within a line must be separated by
+     * a semicolon. The fragment represented by a unique SMILES must be at the very front (first position) of the line.
+     * All other information within the line is ignored.
+     *
```

Comment:
Lines breaks like this are not interpreted when the JavaDoc is created. You can use the <p> HTML tag.